packages/components/src/core/CellBasic/style.ts

```diff
@@ -51,6 +52,7 @@ export const StyledTableData = styled("td", {
   shouldForwardProp: (prop) => !doNotForwardProps.includes(prop as string),
 })`
   ${fontBodyS}
+  ${focusVisibleA11yStyle()}
```

Comment:
There’s a small difference between these two

`fontBodyS` is actually a constant holding the result of another function:
```js
export const fontBodyS = fontBody("s", "regular");
```

On the other hand, `focusVisibleA11yStyle` and `focusA11yStyle` are functions, and we need to invoke them.

Reply:
Thanks for the explanation, Masoud 😄 🙏 

Hmm interesting, I think `fontBodyS` is a function too?

<img width="514" alt="Screenshot 2024-07-03 at 9 07 37 AM" src="https://github.com/chanzuckerberg/sci-components/assets/6309723/c0389c34-12eb-45d5-b566-f89d27898bd2">

<img width="897" alt="Screenshot 2024-07-03 at 9 08 01 AM" src="https://github.com/chanzuckerberg/sci-components/assets/6309723/b45f5d9c-ddb5-45c7-acfc-29b22f749ffc">

Reply:
Oh 😯

I need to review these functions again. It's strange that `fontBodyS` shouldn't be invoked, but `focusVisibleA11yStyle` should be! 🤔

Reply:
I see! So if you just use `${focusVisibleA11yStyle}` we don't get the style in the element?

Reply:
Thats correct, we have to invoke it for some reason! 🤔

Reply:
Huh interesting - I just tested with `${focusVisibleA11yStyle}` and seems to be able to get the focus visible style (I added a pink background to make sure my modification is being picked up 😆 )

<img width="1639" alt="Screenshot 2024-07-08 at 11 11 53 AM" src="https://github.com/chanzuckerberg/sci-components/assets/6309723/258e898a-ca59-49db-b476-8f182bb24ce6">

<img width="774" alt="Screenshot 2024-07-08 at 11 12 20 AM" src="https://github.com/chanzuckerberg/sci-components/assets/6309723/8ef1f392-0a93-432c-9fe2-7f8eb8aa9072">

Reply:
Just like you said, it depends on the context of usage! If it's within another function, it needs to be invoked, otherwise, it doesn't.

Reply:
The issue with the `StyledTableData` component example you've mentioned above, is that the `focusVisibleA11yStyle` doesn't have any effects here! Because the table cell components are not focusable by tab to begin with, so they don't get any focus-visible styles. It was the tooltip component's a11y styles that were showing the outline. If you turn off the tooltip prop for the CellBasic in storybook, you won't see the a11y outline anymore.

Reply:
Ah, forgot to mention that I just removed all the redundant `focusVisibleA11yStyle` calls. The one in the `CellBasic` component got removed too 😯 It was unnecessary.

Reply:
I updated all instances where `focusVisibleA11yStyle` was used outside a function and removed the parentheses. In the example above, as you've already tested, it can be used without parentheses, and there is no need to invoke the function

As for the rest of the instances, where `focusVisibleA11yStyle` is used inside another function, we have to invoke it; otherwise, it won't work.

:D

Reply:
Amazing! Thanks so much for testing and updating them, @masoudmanson 🤩 🙏 Really appreciate your patience on this puzzle 🌟 !! HURRAY!

Reply:
And you're spot on that disabling the tooltip makes the data cell not focusable! 

However, if I add tabIndex="0" to the cell and tab on the cell, I can get the nice black border if I keep `${focusVisibleA11yStyle}` in it, and it'll be the browser blue border if `${focusVisibleA11yStyle}` is left out

```tsx
export const CellBasic = (props: Args): JSX.Element => {
  return (
    <table>
      <tbody>
        <tr
          style={{
            display: "block",
            maxWidth: "400px",
          }}
        >
          <RawCellBasic
            data-testid="CellBasic"
            primaryText="Primary Text"
            tooltipProps={{ sdsStyle: "light" }}
            {...props}
            // added these two 👇
            shouldShowTooltipOnHover={false}
            tabIndex={0}
          />
        </tr>
      </tbody>
    </table>
  );
};

```

<img width="1632" alt="Screenshot 2024-07-09 at 8 33 48 AM" src="https://github.com/chanzuckerberg/sci-components/assets/6309723/688fb210-e696-4225-8e18-1925d001aa37">

<img width="1679" alt="Screenshot 2024-07-09 at 8 34 29 AM" src="https://github.com/chanzuckerberg/sci-components/assets/6309723/5ceb45dc-4f70-46a8-a000-000a69b79e25">

Reply:
so I'm thinking mayybe it might still be worth keeping `${focusVisibleA11yStyle}` in `BasicCell`? What do you think?

Reply:
Ah, you're right. I came to the exact same conclusion last night and I reverted the removed `focusVisibleA11yStyle` functions 😅 
Thank you for checking it over and over again, really appreciate your reviews  @tihuan  🤩🥳😍
I think the PR is ready now 🎉

Reply:
OH! Sorry I totally missed that 😆 Hurray thanks so much again for all the back and forth with me here, @masoudmanson ❤️ LGTM it isss!! 🚀🚀🚀